resources/js/api.ts

```diff
@@ -77,15 +77,21 @@ export async function getGroupCoursesByTerm({
   groupId: number;
   termId: number;
 }) {
-  // const cacheKey: GetGroupByTermCacheKey = `${groupId}-${termId}`;
-  // if (getGroupByTermCache.has(cacheKey)) {
-  //   return getGroupByTermCache.get(cacheKey)!;
-  // }
+  // cache if we're in dev mode (so we don't have to wait for the API)
```

Comment:
Can you add a note about this to the readme? I can see getting tripped up by it at some point in the future.

Reply:
I could see that. I doubt I'll think to look at the readme. I'm thinking maybe we should drop it? It's served it's purpose for now. Or, if we keep it, maybe put it behind a better named env variable like `CACHE_API_RESPONSES_CLIENTSIDE`?  What do you think?

Reply:
I'm fine dropping it